python/paddle/distributed/passes/pass_utils.py

```diff
@@ -444,11 +444,39 @@ def _insert_sync_for_fthenb_1f1b(program):
         block._sync_with_cpp()
 
 
-def _program_for_fthenb_and_1f1b(program):
+def _overlap_send_recv(program):
+    """
+    This function is used to replace the function '_insert_sync_for_fthenb_1f1b'.
+    The finally target of this function is as follows:
+        1. no need to insert the 'c_sync_calc' and 'c_sync_calc' operators
+        2. 'send_v2' operator uses 'dist_attr.execution_stream' to set stream of its own.
+        3. 'recv_v2' opeator uses 'dist_attr.execution_stream' to set stream of its own.
+    """
+    for block in program.blocks:
+        for op in block.ops:
+            if op.type == 'send_v2':
+                op._set_attr("dynamic_shape", False)
+                op._set_attr("use_calc_stream", True)
```

Comment:
use_calc_stream should be false?

Reply:
When `use_calc_stream` is false, `send_v2` will use the streams in`phi::distributed::NCCLCommContext`